slicerio/server.py

```diff
@@ -15,57 +14,73 @@ def start_server(slicer_executable=None, timeoutSec=60):
     import os
     import subprocess
     import time
+
     if not slicer_executable:
-        if 'SLICER_EXECUTABLE' not in os.environ:
-            raise ValueError('SLICER_EXECUTABLE environment variable is not specified')
-        slicer_executable = os.environ['SLICER_EXECUTABLE']
-    p = subprocess.Popen([slicer_executable, "--python-code", f"wslogic = getModuleLogic('WebServer'); wslogic.port={SERVER_PORT}; wslogic.addDefaultRequestHandlers(); wslogic.start()"])
+        if "SLICER_EXECUTABLE" not in os.environ:
+            raise ValueError("SLICER_EXECUTABLE environment variable is not specified")
+        slicer_executable = os.environ["SLICER_EXECUTABLE"]
+    p = subprocess.Popen(
+        [
+            slicer_executable,
+            "--python-code",
+            f"wslogic = getModuleLogic('WebServer'); wslogic.port={SERVER_PORT}; wslogic.addDefaultRequestHandlers(); wslogic.start()",
+        ]
+    )
     start = time.time()
     connected = False
     while not connected:
         connected = is_server_running()
         if time.time() - start > timeoutSec:
-            raise requests.exceptions.ConnectTimeout("Timeout while waiting for application to start")
+            raise requests.exceptions.ConnectTimeout(
+                "Timeout while waiting for application to start"
```

Comment:
I don't like these short lines and unnecessary line breaks, but we can keep these changes and then I'll revert the annoying ones later.

Reply:
it's black... the Uncompromising Code Formatter 😉

Reply:
much less noise without the formatting ...